api/TextMetrics.json

```diff
@@ -602,10 +602,10 @@
               "version_added": false
             },
             "opera": {
-              "version_added": null
+              "version_added": "73"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": false
```

Comment:
I'd be surprised if this API was different between desktop and mobile, can you verify this isn't in Chrome for Android? Then I'd be more inclined to trust this.

Reply:
This _was_ added in Chrome Android, enabled by default in Chrome 87.  There's no Opera Android version that corresponds to Chrome 87, though, hence `false`.

Reply:
Oh, I see, thanks!